mtasa-blue/CODING_GUIDELINES.md

```diff
@@ -147,6 +147,39 @@ and understand its purpose.
         return true;
     }
     ```
+-   Use **early-continue** to improve code readability.
+    ``` cpp
+    // Poor readability
+    bool SomeFunction()
+    {
+        for(auto i=0; i<255; i++)
+        {
+            if(conditionA)
+            {
+                someCode();
+                if(conditionB)
+                {
+                    otherCode();
+                }
+            }
+        }
+    }
+
+    // Clearer readability
+    bool SomeFunction()
+    {
+        for(auto i=0; i<255; i++)
+        {
+            if(!conditionA)
+                continue;
+                
+            someCode();
+            if(!conditionB)
+                continue;
+            otherCode();
```

Comment:
If using brace-less statements, the final line should be separated for readability. e.g:

```cpp
someCode();
if(!conditionB)
    continue;

otherCode();
```

I added a rule for this in cb37729